src/test/kotlin/PersonTest.kt

```
@@ -0,0 +1,23 @@
+import com.survivalcoding.Person
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class PersonTest {
+
+    @Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)